scripts/build_binary.sh

```diff
@@ -30,7 +30,9 @@ mkdir -p $1
 
 cd "${ROOT}"
```

Comment:
(GitHub won't let me comment on the right line)

Can you remove all the commented-out lines starting with `# TODO!!!`?  Your new code replaces that.

Reply:
Removed todo code in 2a7656a